### PR TITLE
Podcast player: Moves podcast-player json schema to podcast helper

### PR DESCRIPTION
**Pull request by creativecoder**
#### Changes proposed in this Pull Request:

Moves json schema for `/podcast-player` endpoint to the podcast helper library, so the schema can be reused in a global wpcom endpoint (see D54402-code), needed for podcast specific onboarding for unauthenticated users in WordPress.com.

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* While editing a post or page, insert the podcast player block
* Input a valid podcast rss feed, and make sure the block works as expected for playing the 5 most recent episodes

#### Proposed changelog entry for your changes:

n/a

### CODE REVIEW

File: _inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-podcast-player.php [hunk 1]
```diff
@@ -16,6 +16,10 @@ class WPCOM_REST_API_V2_Endpoint_Podcast_Player extends WP_REST_Controller {
 	 * Constructor.
 	 */
 	public function __construct() {
+		if ( ! class_exists( 'Jetpack_Podcast_Helper' ) ) {
```
Comment (retrofox) [inline _inc/lib/core-api/wpcom-endpoints/class-wpcom-rest-api-v2-endpoint-podcast-player.php:19]: Just curious. Why did you remove checking the class existence?

**Review by retrofox — APPROVED** [review]
Tested and it works as expected. LGTM.

**Review by jeherve — APPROVED** [review]

**Review by jeherve — APPROVED** [review]
Re-approving after a rebase to fix unrelated test failures.

**Comment by creativecoder** [comment]
r219084-wpcom
